.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2
+jobs:
+  test:
+    working_directory: ~/circle
+    docker:
+      - image: ministryofjustice/ruby:2.5.1
+        environment:
+          RAILS_ENV: test
+          PGHOST: 127.0.0.1
+          PGUSER: root
+          DATABASE_URL: postgresql://root@localhost/fb-user-datastore_test
+      - image: circleci/postgres:10.6-alpine-ram
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: circle-test_test
+    steps:
+      - checkout
+      - run:
+          name: bundle install
+          command: 'bundle install --deployment --jobs=2 --retry 3 --without development'
+      - run:
+          name: setup database
+          command: 'RAILS_ENV=test bundle exec rake db:create db:schema:load'
+      - run:
+          name: test
+          command: 'bundle exec rspec'
+  build:
+    working_directory: ~/circle/git/fb-user-datastore
+    docker:
+      - image: asmega/fb-builder:latest
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "0f:df:74:9e:82:a9:98:7d:0e:fa:be:1a:40:db:77:83"
+      - run:
+          name: checkout fb-user-datastore-deploy
+          command: "GIT_SSH_COMMAND='ssh -v -i ~/.ssh/id_rsa_0fdf749e82a9987d0efabe1a40db7783 -o \"IdentitiesOnly=yes\"' git clone git@github.com:ministryofjustice/fb-user-datastore-deploy.git ~/circle/git/fb-user-datastore-deploy"
+      - run:
+          name: npm install
+          command: "npm install"
```

Comment:
Minor nit: double quotes used here and single quotes used elsewhere

Reply:
fixed